tests/fixture_plugin/utils/route-plugin-multiple-islands.ts

```diff
@@ -0,0 +1,29 @@
+import { Plugin } from "$fresh/server.ts";
+import IslandsPluginComponent from "./sample_routes/PluginRouteWithIslands.tsx";
+import { PluginMiddlewareState } from "$fresh/tests/fixture_plugin/utils/route-plugin.ts";
+
+export default function routePluginMultipleIslands(): Plugin<
+  PluginMiddlewareState
+> {
+  return {
+    name: "routePluginMultipleIslands",
+    routes: [
+      {
+        path: "pluginroutewithislands",
+        component: IslandsPluginComponent,
+      },
+    ],
+    islands: [
+      {
+        baseLocation: import.meta.url,
+        paths: ["./sample_islands/IslandFromPlugin.tsx"],
+      },
```

Comment:
This is interesting because it essentially duplicates the island registered from `route-plugin.ts`. As I was stepping through the code, I saw what was about to happen and kept thinking "no problem, it'll get cleared out later". But it doesn't! But it also doesn't cause problems... 🤔 

I guess it's not a terrible thing to have it like this, because sooner or later someone is bound to end up in a similar situation. I've made note of this in https://github.com/denoland/fresh/pull/1568#issuecomment-1925280649 that we need to detect island conflicts like this. Although this is probably a special case in that the "conflict" isn't an issue, because everything is the same.

Reply:
Any changes to make for this?

Reply:
Nothing to do. I just wanted to highlight that this is happening and isn't causing issues, even though it's unexpected.